torchvision/prototype/features/_segmentation_mask.py

```diff
@@ -1,5 +1,124 @@
+from __future__ import annotations
+
+from typing import Tuple, List, Optional, Union, Sequence
+
+import torch
+from torchvision.transforms import InterpolationMode
+
 from ._feature import _Feature
 
 
 class SegmentationMask(_Feature):
-    pass
+    def horizontal_flip(self) -> SegmentationMask:
+        from torchvision.prototype.transforms import functional as _F
+
+        output = _F.horizontal_flip_segmentation_mask(self)
+        return SegmentationMask.new_like(self, output)
+
+    def vertical_flip(self) -> SegmentationMask:
+        from torchvision.prototype.transforms import functional as _F
+
+        output = _F.vertical_flip_segmentation_mask(self)
+        return SegmentationMask.new_like(self, output)
+
+    def resize(  # type: ignore[override]
+        self,
+        size: List[int],
+        interpolation: InterpolationMode = InterpolationMode.BILINEAR,
```

Comment:
I understand that this is not used but should we have as default value `BILINEAR` which is not really supported for masks? I think it's best if everything changes to NEAREST in this file, to at least indicate that the right value is used.